superbench/config/amd_mi100_hpe.yaml

```diff
@@ -37,6 +37,7 @@ superbench:
   benchmarks:
```

Comment:
Shall we add this feature to document? I think it should be 120 seconds. :)

Reply:
Good idea, will update the docs. I thought amd GPU should have slower kernel launch speed, so enlarge it to 180 seconds.